test/libsolidity/syntaxTests/literalSuffixes/parameters/function_with_signed_exponent.sol

```diff
@@ -0,0 +1,14 @@
+struct Float {
+    uint mantissa;
+    int exponent;
+}
+
+function f(uint mantissa, int exponent) pure returns (Float memory) {
```

Comment:
Can you test also smaller integer types. See e.g. how it's done in `exponent_out_of_range.sol`.

I mean, if possible - if the compiler stops at the first error then this test is fine. But the error is not fatal, so I think you should be able to test multiple functions in a single test case.

Reply:
It is possible. Already done.